composer.json

```diff
@@ -34,6 +34,7 @@
 		"test": "phpunit"
 	},
 	"require-dev": {
-		"scrutinizer/ocular": "^1.3"
+		"scrutinizer/ocular": "^1.3",
+		"phpunit/phpunit": "^5.2"
```

Comment:
Some data I found to support your opinion:
https://seld.be/notes/php-versions-stats-2015-edition
We could offer 5.5+ indeed.